packages/php-wasm/logger/src/logger.ts

```diff
@@ -62,7 +75,19 @@ export class Logger {
 	 * @param PromiseRejectionEvent event
 	 */
 	private logUnhandledRejection(event: PromiseRejectionEvent) {
-		this.log(`${event.reason.stack}`, 'fatal');
+		this.log(`${event.reason.stack}`, 'Error');
+	}
+
+	public maybeDispatchFatalErrorEvent() {
+		if (typeof window !== 'undefined') {
+			window.dispatchEvent(
```

Comment:
If this was dispatched on the logger, as in `this.dispatchEvent()`, the `if` wouldn't be needed and it would also work in non-window contexts like Node.js.

Reply:
Done [a9dd3a9](https://github.com/WordPress/wordpress-playground/pull/1095/commits/a9dd3a9fee9c5f07e9486abb43b606e41ef70ee5)